src/pages/SearchPage.js

```diff
@@ -189,6 +189,7 @@ class SearchPage extends Component {
                                 placeholderText={this.props.translate('optionsSelector.nameEmailOrPhoneNumber')}
                                 onLayout={this.searchRendered}
                                 safeAreaPaddingBottomStyle={safeAreaPaddingBottomStyle}
+                                shouldDelayFocus
```

Comment:
This line was removed in this PR: https://github.com/Expensify/App/pull/10194

According to the description, that line seems to have created a regression in the past: https://github.com/Expensify/App/issues/6069

Can you have a look there and make sure that this won't cause the same regression?

Reply:
I confirmed that [regression](https://github.com/Expensify/App/issues/6069) happens because of the change I made. @aldo-expensify do you have ideas on how we might be able to address the original issue without delaying focusing on text input?

Reply:
No from the top of my head, I'm not familiarized with how we handle focus.

Reply:
I'll seek suggestions in `expensify-open-source` slack channel.

Reply:
Sounds like a good plan ;)